Traceback (most recent call last):
  File "/home/ddlmozco/ddlapplication/passenger_wsgi.py", line 31, in <module>
    application = get_wsgi_application()
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/django/apps/config.py", line 301, in import_models
    self.models_module = import_module(models_module_name)
  File "/opt/alt/python38/lib64/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ddlmozco/ddlapplication/users/models.py", line 61, in <module>
    class User(AbstractUser):
  File "/home/ddlmozco/ddlapplication/users/models.py", line 68, in User
    print(last_active)
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/colorama/ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/colorama/ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/colorama/ansitowin32.py", line 190, in write_and_convert
    self.write_plain_text(text, cursor, len(text))
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/colorama/ansitowin32.py", line 196, in write_plain_text
    self.wrapped.flush()
BrokenPipeError: [Errno 32] Broken pipe
Traceback (most recent call last):
  File "/home/ddlmozco/ddlapplication/passenger_wsgi.py", line 31, in <module>
    application = get_wsgi_application()
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/ddlmozco/virtualenv/ddlapplication/3.8/lib/python3.8/site-packages/django/apps/registry.py", line 83, in populate
    raise RuntimeError("populate() isn't reentrant")
RuntimeError: populate() isn't reentrant
[UID:1823][16196] Child process with pid: 16219 was killed by signal: 15, core dump: 0
2023-07-13 20:00:36.436545